velox/vector/fuzzer/tests/VectorFuzzerTest.cpp

```diff
@@ -688,6 +688,29 @@ TEST_F(VectorFuzzerTest, fuzzRowChildrenToLazy) {
   ASSERT_TRUE(wrappedRow->childAt(1)->as<LazyVector>()->isLoaded());
 }
 
+TEST_F(VectorFuzzerTest, constantOpt) {
+  VectorFuzzer::Options opts;
+  opts.constantRatio = 0.0;
+  opts.dictionaryRatio = 0.0;
+  opts.vectorSize = 100;
+  VectorFuzzer fuzzer(opts, pool());
+
+  bool hasConstant = false;
+  bool hasDictionary = false;
+  for (size_t i = 0; i < 100; ++i) {
+    auto type = fuzzer.randType();
+    auto vector = fuzzer.fuzz(type);
+    if (vector->encoding() == VectorEncoding::Simple::CONSTANT) {
```

Comment:
Why not ASSERT_NE(vector->encoding(), VectorEncoding::Simple::CONSTANT)

Reply:
Fixed